nikola/data/themes/base/templates/index.tmpl

```diff
@@ -11,6 +11,14 @@
 </%block>
 
 <%block name="content">
+%if subcategories:
```

Comment:
add a special block for this to `index.tmpl` for this and use it in `tagindex.tmpl`